### PR TITLE
Added Oracle Linux to the list of platforms.

### PR DESCRIPTION
**Pull request by Djelibeybi**
**\- What I did**

Added Oracle Linux to the list of supported platforms in the FAQ

**\- How I did it**

Modified the docs/faq.md file.

**\- How to verify it**

Read the file

**\- Description for the changelog**

Added Oracle Linux to the list of supported platforms.

**\- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Avi Miller avi.miller@oracle.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
LGTM, but looks like this list is in desperate need of an update (CentOS 6, Ubuntu 13.04, ...)

**Comment by Djelibeybi** [comment]
Yes, I agree. But I don't have authority to change the rest of the list. :) Note that if it is overhauled, Docker does run on both Oracle Linux 6 and Oracle Linux 7.

**Comment by thaJeztah** [comment]
perhaps we should remove this list, and replace it with a link to https://docs.docker.com/engine/installation/ (i.e. `installation/index.md`), which is an up-to-date list, and directly points the user to the relevant installation instructions

**Comment by justincormack** [comment]
This entire page is in need of an update. I think I would delete most of it.

File: docs/faq.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ Linux:
  - Fedora 19/20+
  - RHEL 6.5+
  - CentOS 6+
+ - Oracle Linux 6+
```
Comment (cpuguy83) [inline docs/faq.md]: Doesn't this require UEK for 6? Maybe we can list separately? or add a note.

**Comment by thaJeztah** [comment]
Ok, so I looked at the document as a whole, and agree with @justincormack that we need to clean up this file.

I think it's better to close this PR, and fix the document instead of adding this to a really outdated and incorrect list; I'll open a PR to fix that

**Pull request closed without merging**